src/components/page-intro.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { formatLargeNumber } from '../utils';
+import '../styles/components/page-intro.scss';
+import ChevronDown from '../svg/chevron-down.svg';
+import ChevronUp from '../svg/chevron-up.svg';
+
+class PageIntro extends Component {
+  constructor(props) {
```

Comment:
The `constructor` method here doesn't do anything besides creating the `state` object; Therefore, this can be removed and `state` can be initiated as a class property.

Reply:
Done